whirlpool/dryer.py

```diff
@@ -0,0 +1,288 @@
+import logging
+from enum import Enum
+
+from .appliance import Appliance
+
+LOGGER = logging.getLogger(__name__)
+
+# Machine State
+ATTR_MACHINE_STATE = "Cavity_CycleStatusMachineState"
+
+ATTRVAL_MACHINE_STATE_STANDBY = "0"
+ATTRVAL_MACHINE_STATE_SETTING = "1"
+ATTRVAL_MACHINE_STATE_DELAYCOUNTDOWNMODE = "2"
+ATTRVAL_MACHINE_STATE_DELAYPAUSE = "3"
+ATTRVAL_MACHINE_STATE_SMARTDELAY = "4"
+ATTRVAL_MACHINE_STATE_SMARTGRIDPAUSE = "5"
+ATTRVAL_MACHINE_STATE_PAUSE = "6"
+ATTRVAL_MACHINE_STATE_RUNNINGMAINCYCLE = "7"
+ATTRVAL_MACHINE_STATE_RUNNINGPOSTCYCLE = "8"
+ATTRVAL_MACHINE_STATE_EXCEPTIONS = "9"
+ATTRVAL_MACHINE_STATE_COMPLETE = "10"
+ATTRVAL_MACHINE_STATE_POWERFAILURE = "11"
+ATTRVAL_MACHINE_STATE_SERVICEDIAGNOSTIC = "12"
+ATTRVAL_MACHINE_STATE_FACTORYDIAGNOSTIC = "13"
+ATTRVAL_MACHINE_STATE_LIFETEST = "14"
+ATTRVAL_MACHINE_STATE_CUSTOMERFOCUSMODE = "15"
+ATTRVAL_MACHINE_STATE_DEMOMODE = "16"
+ATTRVAL_MACHINE_STATE_HARDSTOPORERROR = "17"
+ATTRVAL_MACHINE_STATE_SYSTEMINIT = "18"
+ATTRVAL_MACHINE_STATE_MYSTERY = "19" # Observed on "Maytag MGD6230HW3"
+
+# Cavity
+ATTR_OP_STATUS_DOOROPEN = "Cavity_OpStatusDoorOpen"
+ATTR_TIME_STATUS_ESTTIMEREMAINING = "Cavity_TimeStatusEstTimeRemaining"
+ATTR_DISPLAY_SET_DRUMLIGHTON = "Cavity_DisplaySetDrumLightOn"
+
+# DryCavity
+ATTR_CHANGE_STATUS_EXTRAPOWERCHANGEABLE = "Cavity_ChangeStatusExtraPowerChangeable"
```

Comment:
```suggestion
ATTR_EXTRA_POWER_CHANGEABLE = "Cavity_ChangeStatusExtraPowerChangeable"
```
by the way, do you know what does "changeable" mean in this context? should we drop it and make it only `ATTR_EXTRA_POWER`?

Reply:
please also rename the others accordingly

Reply:
Near as I can tell, you should ensure that "Cavity_ChangeStatusExtraPowerChangeable" returns True (1) before attempting to set it with "Cavity_CycleSetExtraPowerEnable".

Reply:
Then `ATTR_EXTRA_POWER_CHANGEABLE` makes sense